config/common.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+/* @var array $params */
+
+use Yiisoft\Aliases\Aliases;
+
+return [
+    Aliases::class => [
+        '__class' => Aliases::class,
+        '__construct()' => [$params['aliases']],
```

Comment:
I think here need use `$params['yiisoft/aliases']['aliases']` and create also params.php with default empty array `aliases`.

Reply:
That's same way as it is now.